builds/linux/rhel/9/variables.pkr.hcl

```diff
@@ -10,8 +10,24 @@
 //  BLOCK: variable
 //  Defines the input variables.
 
+// Additional yum repositories
```

Comment:
```suggestion
// Additional Repositories
```

Reply:
yum_repository should have been in the pull request